src/axom/sidre/core/View.hpp

```diff
@@ -375,6 +375,8 @@ class View
    *       type is NO_TYPE_ID, or ndims < 0, or shape is nullptr,
    *       or any element of shape < 0, this method does nothing.
    *
+   * If shape is NULL, this is a no-op.
```

Comment:
```suggestion
   * If shape is nullptr, this is a no-op.
```
For consistency with other comments